ebzeal/school-manager-backend
# ch(travisCI switch) - Switch to Travis CI

# What does this PR do?
- installs babel and dependencies
- installs coverage, mocha, chai
- creates server
- writes the first test
- adds badges to README

# Description of Task to be completed?
Switch from CircleCI to TravisCI

# How should this be manually tested?
 - clone this branch 

> ch-switch-to-travisCI-166152008
 - run yarn start
- visit http://localhost:3000/
 expected: you should see **"This is School Manager"**
- run yarn test
 expected:  all tests should pass

# Any background context you want to provide?
None

# What are the relevant pivotal tracker stories?
[Switch from CircleCI to TravisCI](https://www.pivotaltracker.com/n/projects/2337013/stories/166152008)

# Story Title
Switch from CircleCI to TravisCI

# Screenshots (if appropriate)
None

# Questions:
None
